opennft/opennft.py

```diff
@@ -2676,7 +2685,7 @@ def drawRoiPlots(self, init):
         dataRaw = np.array(self.outputSamples[key], ndmin=2)[self.selectedRoi,:]
         dataProc = np.array(self.outputSamples['kalmanProcTimeSeries'], ndmin=2)[self.selectedRoi,:]
         dataNorm = np.array(self.outputSamples['scalProcTimeSeries'], ndmin=2)[self.selectedRoi,:]
-        if self.P['PlotFeedback']:
+        if self.P['PlotFeedback'] and self.P['Type'] != 'DCM':
```

Comment:
I guess, it is unnecessary to add this flag if you disable PlotFeedback in `onChangeFBType` method.

I checked the rest and it's fine.